jenkins/workers.py

```diff
@@ -1,144 +0,0 @@
-"""
-Use the jenkins queue and computer APIs to capture info about workers.
```

Comment:
Ah, here's that script I mentioned in the other PR.  Yeah, if this is going away, the shell script over there definitely should also.